### ISSUE TITLE
Improve System Requirements when not Autoscaling

### ISSUE THREAD

**Issue by jtesser** [dotCMS/core#7596]
We need a new property that can turn off the Heartbeat Monitor. When set to off we need to  disable the network Portlet 

In addition we need to not check the assets directory.  ie... we write the heartbeat https://github.com/dotCMS/core/blob/3.2/src/com/dotcms/cluster/business/ServerAPIImpl.java#L87  If someone has a server that is read only this will fail as the assets directory is not writable 

In summary we should shut down all heartbeat stuff when CLUSTER_HEARTBEAT=false is off 
Place the property by the CLUSTER_AUTOWIRE

**Comment by jtesser** [comment]
@oarrietadotcms after done with the current task skype me and lets talk about this one

**Comment by oarrietadotcms** [comment]
Done, PR: https://github.com/dotCMS/core/pull/7618

Notes:

1. The property already existed: ENABLE_SERVER_HEARTBEAT
2. When ENABLE_SERVER_HEARTBEAT is false server will not try to write to disk the Server ID.
3. When ENABLE_SERVER_HEARTBEAT is false Heartbeat Job is disabled (logic was already in place)
4. I moved some Cluster Properties from dotmarketing-config.properties to dotcms-config-cluster.properties

**Comment by oarrietadotcms** [comment]
License PR: https://github.com/dotCMS/license/pull/19

**Comment by oarrietadotcms** [comment]
Pull request to update artifactory

PR for core: https://github.com/dotCMS/core/pull/7697
PR for ee: https://github.com/dotCMS/enterprise-2.x/pull/273

**Comment by bryanboza** [comment]
Fixed, tested on Nightly build 2015-05-12 // H2 // FF
